packages/admin/src/Resources/Pages/EditRecord.php

```diff
@@ -37,14 +37,19 @@ public function getBreadcrumb(): string
     }
 
     public function mount($record): void
+    {
+        $this->authorizeAccess($record);
+
+        $this->fillForm();
+    }
+
+    protected function authorizeAccess($record)
```

Comment:
```suggestion
    protected function authorizeAccess(Model $record): void
```

Reply:
That `$record` refers to the identifier of the model, therefore it would not receive `Model`, but rather int or, failing that, string.